src/middleware/kafkaProducer.js

```diff
@@ -0,0 +1,81 @@
+import logger from 'winston'
+import {Kafka, logLevel} from 'kafkajs'
+
+// Customize Kafka logs
+function kafkaLogger() {
+  const toWinstonLogLevel = level => {
+    switch (level) {
+      case logLevel.ERROR:
+      case logLevel.NOTHING:
+        return 'error'
+      case logLevel.WARN:
+        return 'warn'
+      case logLevel.INFO:
+        return 'info'
+      case logLevel.DEBUG:
+        return 'debug'
+    }
+  }
+  return ({level, log}) => {
+    const {message, ...extra} = log
+    logger[toWinstonLogLevel(level)]({
+      message,
+      extra
+    })
+  }
+}
+
+export class KafkaProducer {
+  static details = {}
```

Comment:
```suggestion
  _options = {}
```
This shouldn't be static, otherwise it will hold a single value across all producers.
Renaming to "options" is better ;)

Reply:
yes 😄 👍🏼 done